### PR TITLE
Update configure-anti-malware-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
added Common attachment type filter and Malware zero hour purge

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/configure-anti-malware-policies.md [hunk 1]
```diff
@@ -60,6 +60,18 @@ Creating a custom anti-malware policy in the EAC creates the malware filter rule
 
      > [!NOTE]
      > If malware is detected in the **message body** of an inbound or outbound message, the entire message is deleted, regardless of the setting you configure for **Malware detection response**.
+     
+   - **Common Attachment Types Filter**: Select one of these values for **blocking attachment types that may harm your computer.**
+   
+     - **Off**
+     
+     - **On**: Emails with attachments of filtered file types will trigger the Malware Detection Response (recommended). Turning this feature On lets you add File types to be blocked.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-anti-malware-policies.md]:
```suggestion
     - **On**: Emails with attachments of filtered file types will trigger the Malware Detection Response (recommended). Turning this feature on lets you add file types to be blocked.
```

File: microsoft-365/security/office-365-security/configure-anti-malware-policies.md [hunk 2]
```diff
@@ -60,6 +60,18 @@ Creating a custom anti-malware policy in the EAC creates the malware filter rule
...
+    
+    - **Malware Zero-hour Auto Purge**: Select one of these values to **Protects your users by automatically taking your policy's action to quarantine messages with malware detected after delivery.**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-anti-malware-policies.md]:
```suggestion
    - **Malware Zero-hour Auto Purge**: Select one of these values to **protect your users by automatically taking your policy's action to quarantine messages with malware detected after delivery.**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #714

Thank you.

cc: @kenwith

**Pull request merged**